code/modules/research/fabricators.dm

```diff
@@ -452,6 +452,11 @@
 		temp = "Unable to connect to local R&D Database.<br>Please check your connections and try again.<br><a href='?src=\ref[src];clear_temp=1'>Return</a>"
 	src.updateUsrDialog()
 
+/obj/machinery/r_n_d/fabricator/kick_act(mob/living/H)
+	..()
+	if(stopped)
```

Comment:
make it check if the user's name is Arthur Herbert Fonzarelli for a 100% success rate